pulsar/README.md

```diff
@@ -0,0 +1,47 @@
+# Apache Pulsar Initialization Action
+
+Here we've got an initialization action that automates the installation of a single-cluster pulsar instance on bare metal. More detail can be found in the Pulsar documentation here: https://pulsar.apache.org/docs/en/deploy-bare-metal/
+
+## Usage
+
+Specify the .sh script's location when using gcloud to create your dataproc cluster. Pulsar requires Zookeeper, which can be installed by either including it as an optional component (https://cloud.google.com/dataproc/docs/concepts/components/zookeeper), or by creating a HA cluster by specifying --num-masters=3.
```

Comment:
s/dataproc/Dataproc

Reply:
Sorry, Dagang. Could you elaborate on this comment?

Reply:
`s/dataproc/Dataproc` means that you need to replace `dataproc` string with `Dataproc`, i.e. capitalize Datarpoc in the readme file where it's not a part of the CLI command.

Reply:
ack. Thanks for the explanation. Done.